commit cf77a512da1ea5c2cfc36e2ba16f05765d1199aa
Author: Rob Johnston <rob.johnston@joyent.com>
Date:   2019-06-18T18:13:33+00:00 (4 months ago)
    
    OS-7026 Add DDI support for Upgradable Firmware Modules
    Reviewed by: Robert Mustacchi <robert.mustacchi@joyent.com>
    Approved by: Jerry Jelinek <jerry.jelinek@joyent.com>

diff --git a/overlay/generic/etc/name_to_major b/overlay/generic/etc/name_to_major
index f63a4cea..557ae7de 100644
--- a/overlay/generic/etc/name_to_major
+++ b/overlay/generic/etc/name_to_major
@@ -305,3 +305,5 @@ ppt 307
 coretemp 308
 amdf17nbdf 309
 pchtemp 310
+ufm 311
+ufmtest 312
